frontend/app/mappers/personal-information-service-mappers.server.ts

```diff
@@ -99,10 +99,10 @@ interface ToUpdateAddressRequestArgs {
   countryId?: string;
   streetName?: string;
   postalCode?: string;
-  provinceTerritoryStateId?: string;
+  id?: string;
```

Comment:
Same comment about renaming the field in `personal-informaton-service-schemas.server`.

Reply:
Fixed, my bad.